docs/usage/configuration-options.md

```diff
@@ -1830,6 +1830,10 @@ Add to this object if you wish to define rules that apply only to patch updates.
 
 Add to this object if you wish to define rules that apply only to PRs that pin dependencies.
 
+## pinDigest
```

Comment:
i think there are some more docs places which need an update to include the new type.

Reply:
I didn't find any other places where I needed to document this new update type.
@HonkingGoose  Can you please assist incase I missed something ?

Reply:
I think we need to add it to `matchUpdateTypes`

Reply:
[Docs for `matchUpdateTypes`](https://docs.renovatebot.com/configuration-options/#matchupdatetypes).

Should we add `pinDigest` to the `allowedValues` array in the table?